mmdet/datasets/pipelines/transforms.py

```diff
@@ -938,3 +938,188 @@ def __call__(self, results):
     def __repr__(self):
         repr_str = self.__class__.__name__ + f'(transforms={self.transforms})'
         return repr_str
+
+
+@PIPELINES.register_module()
+class RandomCenterCropPad(object):
+    """Random center crop and random around padding for CornerNet.
+
+    Args:
+        crop_size (tuple | None): expected size after crop, final size will
+            computed according to ratio. Requires (h, w) in train mode, and
+            None in test mode.
+        ratios (tuple): random select a ratio from tuple and crop image to
+            (crop_size[0] * ratio) * (crop_size[1] * ratio).
+            Only available in train mode.
+        border (int): max distance from center select area to image border.
+            Only available in train mode.
+        mean (tuple): mean value of dataset.
```

Comment:
Could we directly use keys in `img_norm_cfg` by putting this module behind normalize? Or directly use the config's `img_norm_cfg`. If we cannot, we should add some remind here because these hyper-parameters are easy to forget but could affect the performance.

Reply:
I think it's better to do Crop and Resize before Normalize. If the original image has big resolution, the Normalize op will cost lots of time.

To solve this problem, we could set invalid default values to mean, std and to_rgb, and ensure they are assigned with new value in __init__ function.